### DISCUSSION TITLE
Should the `github-actions` `manager` look for `.github` anywhere?

### DISCUSSION THREAD

**Discussion by paddyroddy** [renovatebot/renovate#25770] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

We have a `cookiecutter` template https://github.com/UCL-ARC/python-tooling/tree/main/%7B%7Bcookiecutter.project_slug%7D%7D/.github/workflows where the templates themselves have their own `GitHub Actions` which I'd like to keep updated.

I noticed that `pre-commit` looks for `.pre-commit-config.yaml` anywhere (not just root) which worked out the box for us. Even though this isn't standard behaviour https://docs.renovatebot.com/modules/manager/pre-commit/#default-config.

However, `github-actions` doesn't do this https://docs.renovatebot.com/modules/manager/github-actions/#file-matching.

### What you tried so far.

I have _fixed_ this for myself doing the following https://github.com/UCL-ARC/python-tooling/blob/7ce56ce9305c7d826204019301acfdeabf09609f/.renovaterc.json#L4-L8, but wondered if this should be changed at the `manager` level?

### Relevant debug logs

N/A

**Comment by rarkins** [comment]
There's probably no harm in widening it

**Reply by paddyroddy** [reply, marked as answer]
PR raised #25813
